api/client/newOrgRepo.ts

```diff
@@ -165,7 +165,7 @@ router.post('/repo/:repo', asyncHandler(discoverUserIdentities), asyncHandler(as
   if (!body) {
     return next(jsonError('No body', 400));
   }
-  req.apiVersion = req.query['api-version'] || req.headers['api-version'] || '2017-07-27';
+  req.apiVersion = (req.query['api-version'] || req.headers['api-version'] || '2017-07-27') as string;
```

Comment:
Just to understand, you're getting a build error from this?

Reply:
Yes! I was seeing:

```
api/client/newOrgRepo.ts:168:3 - error TS2322: Type 'string | string[] | ParsedQs | ParsedQs[]' is not assignable to type 'string'.
  Type 'string[]' is not assignable to type 'string'.

168   req.apiVersion = req.query['api-version'] || req.headers['api-version'] || '2017-07-27';
      ~~~~~~~~~~~~~~
```
That's basically the same form the other errors were showing as.